site/en/_data/community.yml

```
@@ -1,40 +1,50 @@
 - logo: ix://landings/home/android-logo.png
+  topic: Android
   cta:
     url: https://android-developers.googleblog.com/
-    text: Android Developer Blog
+    text: Blog
 - logo: ix://landings/home/twitter-logo.png
+  topic: Android
   cta:
     url: https://twitter.com/AndroidDev
-    text: Android Developer Twitter Channel
+    text: Twitter
 - logo: ix://landings/home/youtube-logo.png
+  topic: Android
   cta:
     url: https://www.youtube.com/user/androiddevelopers
-    text: Android Developer YouTube Channel
-- logo: ix://landings/home/twitter-logo.png
-  cta:
-    url: https://twitter.com/ChromiumDev
-    text: Chrome Developer Twitter Channel
+    text: YouTube
 - logo: ix://landings/home/github-logo.png
+  topic: Chrome OS
   cta:
     url: https://github.com/chromeos
-    text: Chrome OS on GitHub
-- logo: ix://landings/home/codelabs-logo.png
-  cta:
-    url: https://developers.google.com/web/fundamentals/codelabs/
-    text: Codelabs & Code Samples
-- logo: ix://landings/home/medium-logo.png
+    text: GitHub
+- logo: ix://landings/home/twitter-logo.png
+  topic: Chrome OS
   cta:
-    url: https://medium.com/googleplaydev
-    text: Google Play on Medium
+    url: https://twitter.com/ChromiumDev
+    text: Twitter
 - logo: ix://landings/home/reddit-logo.png
+  topic: Chrome OS
   cta:
     url: https://www.reddit.com/r/chromeos/
-    text: Chrome OS Reddit Community
+    text: Reddit
 - logo: ix://landings/home/stackoverflow-logo.png
+  topic: General
```

```suggestion
  topic: Chrome OS
```

This SO link goes directly to questions tagged Chrome OS, so the topic should reflect that